src/signals/incident-management/containers/IncidentSplitContainer/styled.js

```diff
@@ -0,0 +1,108 @@
+import styled from 'styled-components';
+
+import { Heading, Label, RadioGroup, themeColor, themeSpacing, Row } from '@datapunt/asc-ui';
+
+import Button from 'components/Button';
+
+export const StyledDefinitionList = styled.dl`
+  margin: 0;
+  display: grid;
+  grid-row-gap: 0;
+
+  @media (min-width: ${({ theme }) => theme.layouts.medium.max}px) {
+    column-gap: ${({ theme }) => theme.layouts.medium.gutter}px;
+    grid-template-columns: 3fr 3fr;
+  }
+
+  @media (min-width: ${({ theme }) => theme.layouts.large.min}px) {
+    column-gap: ${({ theme }) => theme.layouts.large.gutter}px;
+    grid-template-columns: 3fr 2fr;
+  }
+
+  @media (min-width: ${({ theme }) => theme.layouts.large.max}px) {
+    column-gap: ${({ theme }) => theme.layouts.large.gutter}px;
```

Comment:
No need to set `column-gap` here; is already covered by the previous media query